packages/nimble-components/src/rich-text/editor/tests/rich-text-editor.spec.ts

```diff
@@ -890,6 +1038,39 @@ describe('RichTextEditor', () => {
         }
     });
 
+    describe('Should return markdown with back slash (HardBreak) when markdown with hard break syntax are assigned', () => {
```

Comment:
Just curious to understand why you wanted to have this test case now? 

Title suggestion
```suggestion
    describe('`getMarkdown` with hard break backslashes should be same immediately after `setMarkdown`', () => {
```

Reply:
Updated